draft-ietf-taps-interface.md

```diff
@@ -2079,7 +2081,8 @@ Properties will include different information:
   Protocol Selection Properties that the application specified on the
   Preconnection.
 
-* For Connections that are Established: information concerning the
+* For Connections that are Established: Transport Services system implementations
+  ought to provide information concerning the
```

Comment:
Was this update meant to only improve readability or also change the meaning? It now sounds more optional, which may be fine as I am not sure the system will always have this information. Just wanted to check that this was an intentional modification.

Reply:
Just readability: I did this because before, it was a list of things that "are" provided - but the rest of the list is now concrete, assigned to a property, whereas this item isn't (it's more general). I think that, vague as this is, indeed we couldn't make it a "must"... so maybe now it's a bit more explicit that this is indeed optional, which I think is ok.